src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -463,19 +519,35 @@ public CourseDetailsBundle getCourseSummary(String courseId) throws EntityDoesNo
         return getCourseSummary(cd);
     }
     
+    /**
+     * @param instructorAttributes
+     * @return the {@link CourseSummaryBundle course summary}, including
+     *         its feedback sessions using the given {@link InstructorAttributes}.
+     * @throws EntityDoesNotExistException
+     */
     public CourseSummaryBundle getCourseSummaryWithFeedbackSessionsForInstructor(
-            InstructorAttributes instructor) throws EntityDoesNotExistException {
-        CourseSummaryBundle courseSummary = getCourseSummaryWithoutStats(instructor.courseId);
-        courseSummary.feedbackSessions.addAll(feedbackSessionsLogic.getFeedbackSessionListForInstructor(instructor));
+            InstructorAttributes instructorAttributes) throws EntityDoesNotExistException {
```

Comment:
I think we used the shorter form to reduce line length.  When the lines get too long, they need wrapping, and the code gets messier.
Yes, let's keep it consistent with the rest.